cmd/tether/tether.go

```diff
@@ -157,11 +157,17 @@ func run(src extraconfig.DataSource, sink extraconfig.DataSink) error {
 			// check if session has never been started
 			if proc == nil {
 				log.Infof("Launching process for session %s\n", session.ID)
+
+				// launch the session
 				err := launch(session)
+				// save the modified value back to the vmx file
+				extraconfig.EncodeWithPrefix(dataSink, session.Started, fmt.Sprintf(".sessions|%s.started", session.ID))
```

Comment:
would prefer to have this in the launch function and keep this main loop as free from clutter as possible; it's already pretty long.